selfdrive/car/vin.py

```diff
@@ -22,6 +22,7 @@ def get_vin(logcan, sendcan, buses, timeout=0.1, retry=3, debug=False):
         (StdQueries.UDS_VIN_REQUEST, StdQueries.UDS_VIN_RESPONSE, (0, 1), STANDARD_VIN_ADDRS, FUNCTIONAL_ADDRS, 0x8),
         (StdQueries.OBD_VIN_REQUEST, StdQueries.OBD_VIN_RESPONSE, (0, 1), STANDARD_VIN_ADDRS, FUNCTIONAL_ADDRS, 0x8),
         (StdQueries.GM_VIN_REQUEST, StdQueries.GM_VIN_RESPONSE, (0,), [0x24b], None, 0x400),  # Bolt fwdCamera
+        (StdQueries.KWP_VIN_REQUEST, StdQueries.KWP_VIN_RESPONSE, (0,), [0x797], None, 0x3),  # Nissan Leaf unknown ECU
```

Comment:
It would be nice if you could specify requests and rxoffsets per address, it would make any new car-specific queries take no extra time

Reply:
@gregjhogan suggested to use the tester present queries to order the VIN requests, like we do for FW fingerprinting which would also solve this!